spec/domain/people/duplicate_locator_spec.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2012-2020, Jungwacht Blauring Schweiz. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+require 'spec_helper'
+
+describe People::DuplicateLocator do
+
+  let(:checker) { described_class.new }
+
+  context 'check' do
+    it 'creates one duplicate when all attributes match once' do
+      create_duplicate(:top_leader)
+      expect { checker.run }.to change { PersonDuplicate.count }.by(1)
```

Comment:
shouldn't we check at least one time if the generated duplicate entry has the correct persons* assigned ?

* I know ... the plural of person is usually people. BUT: persons is not false ... it's just pretty formal. aaaand, I find it more appropriate in this case :smile: